src/containers/User/SettingsProfile.js

```diff
@@ -29,23 +29,12 @@ export const SettingsProfile = props => {
 
   const makeFormData = (values) => {
     // do not send image data if not changed
-    if (!values['user_image']) {
-      delete values['user_image']
-    } else if (values['user_image'] === profileContext?.newUserImage) {
+    if (!values['user_image'] || !profileContext?.newUserImage
```

Comment:
nice it works! it takes a while to understand all the conditions, though. Maybe remap them to descriptive variables help a bit with readability?

```
    const existingImage = values['user_image']
    const newImage = profileContext?.newUserImage
    const existingIsSameAsNewImage = existingImage === newImage

    if (!existingImage || !newImage || existingIsSameAsNewImage) {
      delete values['user_image']
    } else if (!existingIsSameAsNewImage) {
      // instead of setFieldValue (Formik), assigning "manually" here
      values['user_image'] = profileContext.newUserImage
    }
```